### ISSUE TITLE
Playtest 0.3.39 - Mouse Sensitivity Slider values bug

### ISSUE THREAD

**Issue by CVariable** [loiste-interactive/Obenseuer-Issues#2275]
Upon starting a new game, the mouse sensitivity is fairly high. Opening the menu shows it is set at 1.33:
![grafik](https://github.com/loiste-interactive/Obenseuer-Issues/assets/45665371/7d71030a-a21a-4be3-9a63-b2330d18fd10)
When moving the slider, the value 1.33 suddenly corresponds to a way slower mouse movement than it did before manually modifying the value, aswell as being at a different position.
![grafik](https://github.com/loiste-interactive/Obenseuer-Issues/assets/45665371/c2b3a2ee-e948-428d-b8d1-59839a649809)
Setting it to around 7 is close to what the mouse sensitivity was before moving the slider, while it still said 1.33.
I think this is also the cause of issue #1572.

**Comment by Oskutin** [comment]
Doesn't happen to me, you have controller plugged or something?

**Comment by CVariable** [comment]
No, the only input device plugged in are a corsair keyboard and a redragon (chinese) mouse. If another player encounters this issue next time we should ask them if they happen to use the same device. This bug still happens on every game launch and i could not pinpoint it any further yet
